### ISSUE TITLE
GraphQL query failed - "cannot use map as Boolean"

### ISSUE THREAD

**Issue by rlsnightmare** [gitgiggety/plugin.video.stash#16]
Hi, 
there seems to be an error whenever a saved filter contains the "organized" filter.
The Kodi log says the following:
```
Error Contents: GraphQL query failed:422 - b'{"errors":[{"message":"cannot use map as Boolean","path":["variable","scene_filter","organized"],"extensions":{"code":"GRAPHQL_VALIDATION_FAILED"}}],"data":null}'. Query: 
query findScenes($scene_filter: SceneFilterType, $filter: FindFilterType!) {
  findScenes(scene_filter: $scene_filter, filter: $filter) {
    count
    scenes {
      id
      title
      details
      rating
      date
      created_at
      paths {
        screenshot
      }
      file {
        duration
        video_codec
        audio_codec
        width
        height
      }
      studio {
        name
      }
      performers {
        name
      }
      tags {
        name
      }
      scene_markers {
        id
      }
    }
  }
}
. Variables: {'filter': {'per_page': -1, 'sort': 'created_at', 'direction': 'DESC'}, 'scene_filter': {'organized': {'modifier': 'EQUALS', 'value': 'false'}, 'stash_id_endpoint': {'modifier': 'NOT_NULL', 'value': ''}}}
```
Version 0.0.9

**Comment by level451** [comment]
Im getting the same

**Comment by gitgiggety** [comment]
Thanks for the report and sorry for me not noticing.

I just created a pull request (#21) with some fixes which should make it work again. Could you please test it? You can download a test version [here](https://github.com/gitgiggety/plugin.video.stash/actions/runs/7532034264/artifacts/1170394682), then unpack the zip file, which contains another zip, and this zip resulting zip file can then be installed in Kodi (using "Install from ZIP" option).

**Comment by gitgiggety** [comment]
Fixed in 0.0.10
